templates/admin/notice.tmpl

```diff
@@ -23,7 +23,7 @@
 						<tr>
 							<td class="collapsing">
 								<div class="ui fitted checkbox" data-id="{{.ID}}">
-									<input type="checkbox"> <label></label>
+									<input type="checkbox">
```

Comment:
If I remember correctly, it's done by purpose. (I haven't checked it)

And it's an unrelated change (this PR is improving buttons), so I think it's better to keep it as it was.